packages/jasmine/src/index.js

```diff
@@ -10,3 +10,4 @@ exports.boot = boot;
 // re-export jasmine and its transitive dep jasmine-core
 exports.jasmine = require('jasmine');
 exports.jasmineCore = jasmineCore;
+exports.JUnitXmlReporter = require('jasmine-reporters').JUnitXmlReporter;;
```

Comment:
can you just wrap this with a try/catch to make the reporter optional? Just to make the change less breaking, if we don't find the jasmine-reporters package we could gracefully degrade to the current behavior